tools/xdptrace.wprp

```diff
@@ -88,6 +88,10 @@
             </Keywords>
         </EventProvider>
 
+        <!-- Windows OS providers -->
+        <EventProvider Id="EP_MicrosoftWindowsTcpipPerPacket" Name="2F07E2EE-15DB-40F1-90EF-9D7BA282188A" Level="17" />
+        <EventProvider Id="EP_MicrosoftWindowsWinsockAfd" Name="E53C6823-7BB8-44BB-90DC-3F86090D48A" Level="5" />
```

Comment:
IMHO, `MicrosoftWindows` seems a bit excessive. `Windows` would be plenty, though even that could be omitted.
```suggestion
        <EventProvider Id="EP_TcpipPerPacket" Name="2F07E2EE-15DB-40F1-90EF-9D7BA282188A" Level="17" />
        <EventProvider Id="EP_WinsockAfd" Name="E53C6823-7BB8-44BB-90DC-3F86090D48A" Level="5" />
```

Reply:
These are simply the ETW provider name.